lib/index.js

```diff
@@ -169,16 +169,6 @@ Segment.prototype.initialize = function() {
     self.ready();
   });
 
-  // Migrate from old cross domain id cookie names
-  if (this.cookie('segment_cross_domain_id')) {
```

Comment:
I removed this because we validated that legacy cookies are not used anymore, and it simplifies the logic later in this file to only check for the `seg_xid_ts` cookie.